tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarClusterTestBase.java

```diff
@@ -57,25 +57,40 @@
         };
     }
 
+    @DataProvider(name = "ServiceAndAdminUrls")
+    public static Object[][] serviceAndAdminUrls() {
+        return new Object[][] {
+            // plain text
+            {
+                pulsarCluster.getPlainTextServiceUrl(),
+                pulsarCluster.getHttpServiceUrl()
+            }
+        };
+    }
+
     protected static PulsarCluster pulsarCluster;
 
     @BeforeClass
     public void setupCluster() throws Exception {
-        this.setupCluster("");
+        setupClusterNoWorker();
         pulsarCluster.startFunctionWorkersWithProcessContainerFactory(1);
     }
 
-    public void setupCluster(String namePrefix) throws Exception {
+    protected void setupClusterNoWorker() throws Exception {
```

Comment:
Does any of this work stuff need to be in this PR? It looks unrelated. If so, move it to another patch.

Reply:
We need to make the name explicit it's confusing whether there is worker or not since it's not part of the spec